gcp/modules/nginx-ingress/values.yaml

```diff
@@ -9,9 +9,23 @@ controller:
     maxReplicas: 4
     targetCPUUtilizationPercentage: 50
     targetMemoryUtilizationPercentage: 50
+  resources:
```

Comment:
Is there anything here that should go to the upstream chart?

Reply:
This is release values yaml, not a part of the chart.

Reply:
Right, just checking.